samples/spring-hello-world-action/README.md

```diff
@@ -0,0 +1,51 @@
+# hello-world
+
+
+
+To understand the Kalix concepts that are the basis for this example, see [Designing services](https://docs.kalix.io/services/development-process.html) in the documentation.
+
+
+
+This project contains the framework to create a Kalix application by adding Kalix components. To understand more about these components, see [Developing services](https://docs.kalix.io/services/). Spring-SDK is an experimental feature and so far there is no [official](https://docs.kalix.io/) documentation. Examples can be found [here](https://github.com/lightbend/kalix-jvm-sdk/tree/main/samples) in the folders with "spring" in their name.
+
+
+
+Use Maven to build your project:
+
+```shell
+mvn compile
+```
+
+
+
+To run the example locally, you must run the Kalix proxy. The included `docker-compose` file contains the configuration required to run the proxy for a locally running application.
+It also contains the configuration to start a local Google Pub/Sub emulator that the Kalix proxy will connect to.
+To start the proxy, run the following command from this directory:
+
+```shell
+docker-compose up
+```
+
+To start the application locally, the `exec-maven-plugin` is used. Use the following command:
+
+```shell
+mvn spring-boot:run
+```
+
+With both the proxy and your application running, once you have defined endpoints they should be available at `http://localhost:9000`. 
+
```

Comment:
not sure if you're still going ahead with this sample.. is so, maybe we could add one example of usage here.

Reply:
Yep, I will change to a Counter Value Entity, but indeed we need a call to demonstrate it.